### ISSUE TITLE
Backing up multiple DBs with wal-e

### ISSUE THREAD

**Issue by brdude** [wal-e/wal-e#267]
I'm having some issues backing up multiple DBs from 1 server, and restoring the DBs to a respective server per DB.

The 1st DB living on the server restores and starts with no issue. However the 2nd DB after being restored always failst to start with the following error.

>  * Starting PostgreSQL 9.4 database server  
>  * The PostgreSQL server failed to start. Please check the log output:  
> 2016-08-11 03:00:29 UTC [21539-1] LOG:  database system was interrupted; last known up at 2016-08-10 13:38:22 UTC  
> 2016-08-11 03:00:29 UTC [21540-1] [unknown]@[unknown] LOG:  incomplete startup packet  
> 2016-08-11 03:00:29 UTC [21539-2] LOG:  creating missing WAL directory "pg_xlog/archive_status"  
> 2016-08-11 03:00:29 UTC [21541-1] postgres@postgres FATAL:  the database system is starting up  
> 2016-08-11 03:00:29 UTC [21539-3] LOG:  entering standby mode  
> 2016-08-11 03:00:29 UTC [21539-4] LOG:  invalid primary checkpoint record  
> 2016-08-11 03:00:29 UTC [21539-5] LOG:  invalid secondary checkpoint record  
> 2016-08-11 03:00:29 UTC [21539-6] PANIC:  could not locate a valid checkpoint record  
> 2016-08-11 03:00:29 UTC [21545-1] postgres@postgres FATAL:  the database system is starting up  
> 2016-08-11 03:00:30 UTC [21548-1] postgres@postgres FATAL:  the database system is starting up  
> 2016-08-11 03:00:30 UTC [21551-1] postgres@postgres FATAL:  the database system is starting up  
> 2016-08-11 03:00:31 UTC [21536-1] LOG:  startup process (PID 21539) was terminated by signal 6: Aborted  
> 2016-08-11 03:00:31 UTC [21536-2] LOG:  aborting startup due to startup process failure  
>    ...fail!  

My setup, is the following:
Existing production server with the following 2 DBs
/db0/main/
/db1/main/

Each backing up to their respective bucket in S3.
Trying to restore each DB to be on it's own server (VM) to act as a backup. So:

ServerDB0: /db0/main/
ServerDB1: /db1/main/

When restoring the db0 server starts up with no issues. However db1 always fails with the errors I pasted above.

The one very odd thing I have noticed is that the *_backup_stop_sentinel.json file for DB1 seems to reference the pg_xlog files of DB0. Bellow are the *_backup_stop_sentinel.json for both DBs.

DB0:
`{"wal_segment_backup_stop": "00000001000157B10000000A", "expanded_size_bytes": 1090483815562, "wal_segment_offset_backup_stop": "09533160", "spec": {"tablespaces": [], "base_prefix": "/db0/main/"}}`

DB1:
`{"wal_segment_backup_stop": "00000001000157CC000000CF", "expanded_size_bytes": 1074356008036, "wal_segment_offset_backup_stop": "13969968", "spec": {"tablespaces": [], "base_prefix": "/db1/main/"}}`

Notice the wal_segment_backup_stop on both of these, both of those exist under /db0/main/pg_xlog/archive_status with a .done extension. However Neither of those exists under /db1/main/pg_xlog/archive_status

So in conclusion does wal-e support backing up multiple DBs living on the same server?

**Comment by fdr** [comment]
You can use PGHOST/PGPORT/et al to fix this. wal-e runs `psql` to pick up defaults of the system, which are liable to be a mismatch at times if one is running multiple servers on the same system.
